NextcloudTalk/NCRoom.m

```diff
@@ -233,13 +235,14 @@ - (BOOL)isLeavable
     // (No need to check room type because in one2one rooms users will always be moderators)
     // or when in a group call and there are other participants.
     // We can also check "canLeaveConversation" since v2
-    return self.canLeaveConversation || ![self canModerate] || (self.type != kNCRoomTypeOneToOne && [self.participants count] > 1);
+    return self.canLeaveConversation || ![self canModerate] || (self.type != kNCRoomTypeOneToOne && [self.participants count] > 1)
+        || (self.type != kNCRoomTypeFormerOneToOne && [self.participants count] > 1);
```

Comment:
A former 1:1 would never have a participant count > 1 ?!
But also, the check for delete room fails: https://github.com/nextcloud/talk-ios/blob/49dbc7aa9184bfa342200d4ec0e9ac7c90a6e504/NextcloudTalk/RoomInfoTableViewController.m#L477-L479 because canModerate checks for a locked 1:1.